sys-fs/cryfs/cryfs-0.9.7.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
We always want to use the newest EAPI available. This may expose newer API in eclasses, get rid of some deprecated bits but most importantly it gets us one step closer to cleaning up old EAPIs. While 5-6 indeed isn't that significant, a few years from now we'd like to avoid having to put more effort into cleaning up packages that were already added with old EAPI.